app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,854 @@
+	<string name="msg_play_added">Zapisano Twoją <b>%1$s</b> rozgrywkę</string>
+	<string name="msg_play_updated">Zaktualizowano rozgrywkę</string>
+	<string name="msg_play_deleted">Usunięto rozgrywkę</string>
+	<string name="msg_play_deleted_draft">Usunięto wersję roboczą</string>
+	<string name="msg_play_update_bad_id">Rozgrywka <b>%s</b> nie istnieje!</string>
+	<string name="msg_play_upload"><b>%1$s</b> - %2$s</string>
+	<string name="msg_play_update_null_response">Błąd komunikacji z BGG.</string>
+	<string name="msg_play_player_change">Zmieniono nazwę gracza z \"%1$s\" na \"%2$s\"</string>
+	<string name="msg_player_add_username">Dodano nazwę użytkownika \"%1$s\" do gracza \"%2$s\"</string>
+	<string name="msg_play_location_change">Zmieniono lokalizację z \"%1$s\" na \"%2$s\"</string>
+	<string name="msg_play_sync">Obecnie nie synchronizujesz swoich rozgrywek. Włączyć tę funkcję?</string>
+	<string name="msg_collection_view_deleted">Pomyślnie usunięto widok kolekcji</string>
+	<string name="msg_saved">Zapisano!</string>
+	<string name="msg_sign_out_success">Pomyślnie wylogowano!</string>
+	<string name="msg_warning_private_info">Uwaga! Jakiekolwiek zmiany tutaj spowodują usunięcie daty inwentarza.</string>
+	<string name="title_share">Udostępnij</string>
+	<string name="title_share_game">Udostępnij grę</string>
+	<string name="title_share_games">Udostępnij gry</string>
+	<string name="title_share_collection">Udostępnij kolekcję</string>
+	<string name="share_collection_subject">Kolekcja należąca do %1$s (%2$s) w BGG4Android</string>
+	<string name="share_collection_complete_footer">Pełna kolekcja na %s</string>
+	<string name="share_game_subject">%s na BoardGameGeek</string>
+	<string name="share_game_text">Znalazłem tę grę w aplikacji BoardGameGeek on Android i myślę, że Ci się spodoba:</string>
+	<string name="share_games_subject">Gry na BoardGameGeek</string>
+	<string name="share_games_text">Znalazłem te gry w aplikacji BoardGameGeek on Android i myślę, że Ci się spodobają:</string>
+	<string name="share_play_title">Udostępnij rozgrywkę</string>
+	<string name="share_thread_subject">Wątek na forum BoardGameGeek</string>
+	<string name="share_thread_text">Znalazłem wątek \"%1$s\" na forum \"%2$s\" w aplikacji BoardGameGeek on Android.</string>
+	<string name="share_thread_game_text">Znalazłem wątek \"%1$s\" na forum \"%2$s\" gry \"%3$s\" w aplikacji BoardGameGeek on Android.</string>
+	<string name="share_thread_article_text">Znalazłem artykuł \"%1$s\" na forum \"%2$s\" w aplikacji BoardGameGeek on Android.</string>
+	<string name="share_thread_article_object_text">Znalazłem artykuł \"%1$s\" na forum \"%2$s\" gry \"%3$s\" w aplikacji BoardGameGeek on Android.</string>
+	<string name="share_geeklist_subject">GeekLista na BoardGameGeek</string>
+	<string name="share_geeklist_text">Znalazłem GeekListę \"%1$s\" w aplikacji BoardGameGeek on Android.</string>
+	<string name="timer">Zegar</string>
+	<string name="times">times</string>
```

Comment:
This is not used anywhere in code (I can't find), so I'm not sure of the context. (Btw there is also a plural down the code).

Reply:
```suggestion
	<string name="times">razy</string>
```